### DISCUSSION TITLE
What is a basic example of the remote config ? Not sure if am doing it correctly

### DISCUSSION THREAD

**Discussion by dbrax** [spatie/laravel-remote#10] [Q&A]
Hello spatie and team,

This is a useful package from you guys am sure it will help me alot.

I am getting this error  though 

  Spatie\Remote\Config\RemoteConfig::getHost(): Argument #1 ($hostName) must be of type string, null given, called in /Users/emmanuel/Desktop/Projects/laravelapps/tujengeportal/vendor/spatie/laravel-remote/src/Commands/RemoteCommand.php on line 21

  at vendor/spatie/laravel-remote/src/Config/RemoteConfig.php:9
      5▕ use Spatie\Remote\Exceptions\CouldNotExecuteCommand;
      6▕
      7▕ class RemoteConfig
      8▕ {
  ➜   9▕     public static function getHost(string $hostName): HostConfig
     10▕     {
     11▕         $configValues = config("remote.hosts.{$hostName}");
     12▕
     13▕         if (is_null($configValues)) {

      +14 vendor frames
  15  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

Just wanted to know if i am using the package correctly 

This is my env

REMOTE_HOST="unleashedafrica.com"
REMOTE_PORT=22
REMOTE_USER=unleashed
REMOTE_PATH=/home/unleashed/uwezo

and this is my remote config file

<?php

return [

    /*
     * This host will be used if none is specified
     * when executing the `remote` command.
     */
    'default_host' => 'default',

    /*
     * Here you can define the hosts where the commands should be executed.
     */
    'hosts' => [
        'default' => [
            'host' => env('REMOTE_HOST',"unleashedafrica.com"),

            'port' => env('REMOTE_PORT', 22),

            'user' => env('REMOTE_USER'),

            /*
             * The package will cd to the given path before executing the given command.
             */
            'path' => env('REMOTE_PATH'),
        ]
    ],
];

Am i using the package correctly or am i missing something ??

**Comment by sinnbeck** [comment]
Try clearing your cache

```
php artisan optimize:clear
